appveyor.yml

```diff
@@ -9,6 +9,9 @@ install:
 build_script:
     - cmd: build.netstd.cmd
 
+nuget:
+    disable_publish_on_pr: true

```

Comment:
Checks "Do not publish NuGet package artifacts to account/project feeds on Pull Requests" on https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/settings/nuget